code/datums/components/z_parallax.dm

```diff
@@ -0,0 +1,55 @@
+/**
+ * Component that hooks into the client, listens for COMSIG_MOVABLE_Z_CHANGED, and depending on whether or not the
+ * Z-level has ZTRAIT_NOPARALLAX enabled, disable or reenable parallax.
+ */
+
+/datum/component/zparallax
+	dupe_mode = COMPONENT_DUPE_UNIQUE
+
+	/// If TRUE, disable parallax checking alltogether until we change Z-levels again.
+	var/caching = FALSE
+
+	var/client/tracked
+	var/mob/atomic
```

Comment:
This name makes zero sense to me.

Reply:
you have found the relic from when i was still typing as atom/movable woops